### PR TITLE
Add terminate notification profile and scheduled events profile 

### PR DESCRIPTION
**Pull request by szimmer-dev**
Add terminate notification profile and scheduled events profile to the Microosft.Compute autorest config.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 1]
```diff
@@ -9241,6 +9245,27 @@
...
+        }
+      }
+    },
+    "TerminateNotificationProfile": {
+      "properties": {
+        "notBeforeTimeout": {
+          "type": "string",
+          "description" : "Configurable length of time a Virtual Machine being deleted will have to potentially approve the Terminate Scheduled Event before the event is auto approved (timed out). The configuration must be specified in ISO 8601 format, the default value is 5 minutes (PT5M)"
+        },
+        "enable" : {
```
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: Is this optional?  What is the default value if this property is not given?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 2]
```diff
@@ -9241,6 +9245,27 @@
...
+          "type": "boolean",
+          "description": "Specifies whether the Terminate Scheduled event is enabled or disabled."
+        }
+      }
+    },
```
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: If both notBeforeTimeout and enable are optional, is terminateNotificationProfile: {} allowed?   If so, what is the meaning of it?  (What is the difference between not giving terminateNotificationProfile and giving {} for terminateNotificationProfile ?  Can we remove terminateNotificationProfile  during update if {} is given for update?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json (hunk 1 above)
Comment (szimmerMicrosoft) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: The description I wrote says the default is 5 minutes

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 3]
```diff
@@ -5483,7 +5483,7 @@
           "items": {
             "type": "string"
           },
-          "description": "Availability Zone to use for this host group � only single zone is supported. The zone can be assigned only during creation. If not provided, the group supports all zones in the region. If provided, enforces each host in the group to be in the same zone."
+          "description": "Availability Zone to use for this host group � only single zone is supported. The zone can be assigned only during creation. If not provided, the group supports all zones in the region. If provided, enforces each host in the group to be in the same zone."
```
[the remark this answers is not in the document]
Comment (szimmerMicrosoft) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]:
```suggestion
          "description": "Availability Zone to use for this host group only single zone is supported. The zone can be assigned only during creation. If not provided, the group supports all zones in the region. If provided, enforces each host in the group to be in the same zone."
```

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json (hunk 1 above)
Comment (hyonholee) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: I mean, the default value for the Boolean variable "enable".

Comment (szimmerMicrosoft) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: Ah default for enable is false

**Pull request closed without merging**